lib/DDGC/Config/Subscriptions.pm

```diff
@@ -19,18 +19,27 @@ sub created_ia_page {
     };
 }
 
+sub created_ia_page {
+    my ( $self, $meta ) = @_;
+    $self->_subscription(
+        'instant_answer',
+        'created',
+        $meta,
+    );
+}
+
 sub updated_ia_page {
     my ( $self, $meta ) = @_;
-    +{
-        category     => 'instant_answer',
-        action       => 'updated',
-        meta1        => $meta->{meta1} // undef,
-        meta2        => $meta->{meta2} // undef,
-        meta3        => $meta->{meta3} // undef,
-        ( $meta->{description} )
-            ? ( description  => $meta->{description} )
-            : (),
-    };
+    $self->_subscription(
+        'instant_answer',
+        'updated',
+        $meta,
+    );
+}
+
+sub ia_page {
+    my ( $self, $id ) = @_;
+    $self->updated_ia_page( { meta1 => $id } );
```

Comment:
Yeah, I fumbled around with this too.

Entity / ID links are common enough in the project and work well enough, but I think I eschewed that here since we're not actually linking to entities / creating relationships. We are just describing the event parameters. It should be equally usable for events within and without the system.

*shrug* We're not so tied to anything that we can't change it.